### PR TITLE
Fix lint issue from context.WithValue

### PR DESCRIPTION
**Pull request by vancexu**

### CODE REVIEW

File: internal/internal_activity.go [hunk 1]
```diff
@@ -105,12 +105,15 @@ type (
...
+
+	// context.WithValue need this type instead of basic type string to avoid lint error
+	contextKey string
 )
 
 const (
-	activityEnvContextKey          = "activityEnv"
-	activityOptionsContextKey      = "activityOptions"
-	localActivityOptionsContextKey = "localActivityOptions"
+	activityEnvContextKey          contextKey = "activityEnv"
```
Comment (mfateev) [inline internal/internal_activity.go]: Does this syntax makes activity and localActivity keys of contextKey type as well?

Comment (vancexu) [reply internal/internal_activity.go]: nop, only activityEnvContextKey is of contextKey

Comment (vancexu) [reply internal/internal_activity.go]: I changed their order to make it looks even safer

File: internal/internal_activity.go [hunk 2]
```diff
@@ -105,12 +105,15 @@ type (
...
+	contextKey string
 )
 
 const (
-	activityEnvContextKey          = "activityEnv"
-	activityOptionsContextKey      = "activityOptions"
-	localActivityOptionsContextKey = "localActivityOptions"
+	activityOptionsContextKey                 = "activityOptions"
+	localActivityOptionsContextKey            = "localActivityOptions"
+	activityEnvContextKey          contextKey = "activityEnv"
```
Comment (yiminc) [inline internal/internal_activity.go]: why not change for the other 2 as well?

**Comment by Groxx** [comment]
why not change them all?  lint is warning about using "normal" key types for context because they're able to collide with other code (including by accident).

**Pull request merged**